scout/server/blueprints/variants/static/form_scripts.js

```diff
@@ -123,6 +127,7 @@ function initSearchConstraints(selectorId, textId){
     });
 }
 
+/* exported enableDismiss */
```

Comment:
Will it be recognised by Lint / CodeFactor? I used the form from there. What do you think about just making the actual function definition "export function enableDismiss"? Would it break anything?

Reply:
Isn't this a comment?

Reply:
Yes, but specifically crafted for lint and CodeFactor to recognise a javascript variable or function intended for use in another file:
> [CodeFactor](https://www.codefactor.io/repository/github/clinical-genomics/scout/overview/master) found an issue: &#39;eraseChromPosString&#39; is defined but never used.
>
> It's currently on:
[scout\server\blueprints\variants\static\form_scripts.js:155](https://www.codefactor.io/repository/github/clinical-genomics/scout/source/master/scout/server/blueprints/variants/static/form_scripts.js#L155)
Commit b1e9c65c366dff96e5ff0d3fb741147553ba1416

Reply:
Ahhh that one! Were it for me, for comments like these CodeFactor could die in a loop of eternal sorrow. But feel free to change it as you want! 🉑

Reply:
Heh, yes, that is not terribly important. An actual change in the language might be nice, but it is a tad weird when the importing is embedded in html. I would blame the lint definition writers rather than CodeFactor. There is probably a config file somewhere where we could ignore certain lint codes, but comments seemed a simple way to get rid of the only "C-level" file we had.